freqtrade/rpc/rpc.py

```diff
@@ -247,7 +248,8 @@ def _rpc_status_table(self, stake_currency: str,
                     trade.id,
                     trade.pair + ('*' if (trade.open_order_id is not None
                                           and trade.close_rate_requested is None) else '')
-                               + ('**' if (trade.close_rate_requested is not None) else ''),
+                               + ('**' if (trade.close_rate_requested is not None) else '')
+                               + f' direction_str',
```

Comment:
I think we should show if the trade is long / short in the status table command. Just need to make it "prettier"

Reply:
for "/status" - it might be pretty easy.
`/status table` might be harder - maybe prefixing the trade id with "L" or "S"? (so it's S-2 - or L-5) ... 
it's not "too pretty" - but i'd not know how to show this better

Reply:
I went with "2 L" or "3 S"